src/vmm/src/devices/virtio/net/device.rs

```diff
@@ -197,6 +199,8 @@ impl Net {
             activate_evt: EventFd::new(libc::EFD_NONBLOCK).map_err(NetError::EventFd)?,
             mmds_ns: None,
             metrics: NetMetricsPerDevice::alloc(id),
+            // SAFETY: Only constructed in the VMM thread so no concurrent buffers
```

Comment:
I don't think we need to have a `SAFETY` comment for this. calling `IoVecBuffer::default()` is not `unsafe`

Reply:
same as before

Reply:
Yup, I think these might be left-overs from when `new` was `unsafe`?

Reply:
Ah yes must of been left over my bad